### PR TITLE
Presence addition + Changes to existing presence

### PR DESCRIPTION
**Pull request by alexbcberio**
Ok, this is gonna be a big one.

### Jellyfin presence
 - When playing video media it's check the info of it to show the most appropiate presence (Movie, Tv-Show or LiveTV)
 - Added presence for music playback and music libraries
 - Implemented presence of LiveTV
 - Code cleaning and some bug fixes

Jellyfin music playback

### Emby presence (*new*)
 - New presence
 - Supports both, the official site and the web client
 - Display what are you playing on the web client: movies, tv-series, music.
 - Display interactions with the web app, on home, on admin panel, browsing data of an item

Emby on official website
Emby web app interaction
Emby watching movie
Emby watching tv-series
Emby playing back music

### CODE REVIEW

File: Emby/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "id": "202915432175239169"
+  },
+  "url": "emby.media",
+  "description": {
+    "en": "Your personal media on any device. Bringing all of your home videos, music, and photos together into one place has never been easier. Your personal Emby Server automatically converts and streams your media on-the-fly to play on any device."
+  },
+  "service": "Emby",
+  "version": "1.0",
+  "logo": "https://imgur.com/download/mTHBDPm",
+  "thumbnail": "https://imgur.com/download/a80aJAx",
```
Comment (Alanexei) [inline Emby/dist/metadata.json]: These must be `i.imgur` links.

File: Emby/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "color": "#52B54B",
+  "category": "videos",
+  "tags": [
+      "video",
+      "movie",
+      "tv",
+      "media"
+  ],
+  "regExp": "([a-z0-9]+[.])*([a-z0-9]+)"
```
Comment (Alanexei) [inline Emby/dist/metadata.json]: Don't include a regExp that selects everything, best to just remove this.

Comment (Bas950) [reply Emby/dist/metadata.json]: @Alanexei No thats needed for this, if this only actives on a certain if its all good

**Comment by Bas950** [comment]
Just a tip: 
For your console logs you might want to use these:
`function PMD_info(message) {
  console.log(
    "%cPreMiD%cINFO%c " + message,
    genericStyle + "border-radius: 25px 0 0 25px; background: #596cae;",
    genericStyle + "border-radius: 0 25px 25px 0; background: #5050ff;",
    "color: unset;"
  );
}

function PMD_error(message) {
  console.log(
    "%cPreMiD%cERROR%c " + message,
    genericStyle + "border-radius: 25px 0 0 25px; background: #596cae;",
    genericStyle + "border-radius: 0 25px 25px 0; background: #ff5050;",
    "color: unset;"
  );
}

function PMD_success(message) {
  console.log(
    "%cPreMiD%cSUCCESS%c " + message,
    genericStyle + "border-radius: 25px 0 0 25px; background: #596cae;",
    genericStyle +
      "border-radius: 0 25px 25px 0; background: #50ff50; color: black;",
    "color: unset;"
  );
}`

**Comment by Alanexei** [comment]
Maybe try making some of the content in your `details` and `state` capitalized on the Emby presence. Looks nicer overall.

**Comment by alexbcberio** [comment]
> Maybe try making some of the content in your `details` and `state` capitalized on the Emby presence. Looks nicer overall.

The only thing that is not capitalized is the "web client" details, everything else is capitalized. I didn't capitalize it on purpose, so it doesn't stand out so much.

Should I capitalize it?

File: Emby/dist/metadata.json (hunk 1 above)
Comment (alexbcberio) [reply Emby/dist/metadata.json]: Update in commit [e4e9b9f78234833a1a64cb3093f3ee71fc0cc253](https://github.com/alexbcberio/Presences/commit/e4e9b9f78234833a1a64cb3093f3ee71fc0cc253)

**Comment by Bas950** [comment]
Tbh it stands out that it isn't capitalized xD

**Comment by alexbcberio** [comment]
Sorry for being a bit stubborn, proves of the changes:
Jellyfin

Emby

**Pull request merged**